ansible_runner/utils/__init__.py

```diff
@@ -27,19 +27,21 @@
 from six import string_types, PY2, PY3, text_type, binary_type
 
 
-def _cleanup_folder(folder):
+def cleanup_folder(folder):
+    """Deletes folder, returns True or False based on whether a change happened."""
     try:
         shutil.rmtree(folder)
```

Comment:
For systems that we care about, it seems that rmtree() (by default) will NOT follow symlinks. Which I like because security.

https://docs.python.org/3/library/shutil.html#shutil.rmtree.avoids_symlink_attacks